runtime/Cargo.toml

```diff
@@ -36,7 +36,7 @@ sha2 = "0.10"
 fvm_sdk = { version = "3.0.0-alpha.2", optional = true }
 
 # test_util
-rand = { version = "0.8.5", default-features = false, optional = true }
+rand = { version = "0.8.5", default-features = false, optional = true, features = ["std_rng"] }
```

Comment:
This won't fly in a WASM environment.

Reply:
wait, this is a pseudo random number generator.
And we need it, else the test_utils thing doesnt compile at all.

Reply:
also, it should be tree shaken away in the bundle, it is only used by test utils.

Reply:
Ah, for a minute I thought it was an environment-dependent RNG, which is prone to introduce a dependency on wasm-bindgen. This doesn't, so we're good (CI checks are green).

Reply:
But maybe i can do it with a conditional feature when we enable test utils?

Reply:
Hm, unfortunately tree shaking wouldn't have helped here. Features enable conditional compilation, which in the past has dragged along wasm-bindgen deps.

Reply:
@vyzo I like introducing a testing feature. Ideally the testing utils would just not be in this crate. But we also don't want to diverge too much from master in common components like this. A "testing" feature would be great, which the actors can enable in dev-dependencies?

Reply:
yeah, let me try that real quick.

Reply:
https://github.com/filecoin-project/builtin-actors/pull/648/commits/b9f1557d51b5703e9dc3d5931ef0f74f3cb65a71 seems to do it, let's wait for CI to green first however.

Reply:
ok, we are green; merging.